internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -28,7 +28,7 @@ services:
       - "127.0.0.1:9200:9200"
 
   elasticsearch_is_ready:
-    image: tianon/true:multiarch
+    image: "${IS_READY_IMAGE_REF}"
```

Comment:
Should it be updated also the docker-compose for local services ? 
https://github.com/elastic/elastic-package/blob/f00e430124b1e8bb43f234fcc7e756461a107786/internal/stack/_static/local-services-docker-compose.yml.tmpl

To use that environment variable, this file uses facts instead. Here it is the method where those facts are set:
https://github.com/elastic/elastic-package/blob/f00e430124b1e8bb43f234fcc7e756461a107786/internal/stack/localresources.go#L55